firmware/src/board/ublox.hpp

```diff
@@ -278,6 +278,7 @@ struct Config
 {
     float fix_rate_hz = 10;
     float aux_rate_hz = 1;
+    uint32_t nav_filter_model = static_cast<uint32_t>(msg::CFG_NAV5::DynModel::Airborne_4g);
```

Comment:
One should take advantage of the C++ type system whenever possible, as that results in safer and more reliable code. Here I would suggest to replace `uint32_t` with `msg::CFG_NAV5::DynModel`. Type casts in other places will need to be updated accordingly.